### DISCUSSION TITLE
How many characters can ALS-Refactored can support?

### DISCUSSION THREAD

**Discussion by Bigbossbro08** [Sixze/ALS-Refactored#389] [General]
I'm doing Mount And Blade like game and I'm thinking of using ALS-Refactored for real. I tried community version but that version is expensive just using characters around 100 ish. I'm wondering does ALS-Refactored did something in that regard?

**Comment by xkamronx** [comment]
ALS Refactored is significantly better optimized when you have 100 players. It's very well optimized overall, and works basically flawlessly. There are a few things you can do based on how many players are on screen etc, but it starts to get funky if you attempt to overoptimize it (like the animations not playing at distance if using the auto net optimize feature in Unreal).

For the most part, ALS Refactored likely isn't going to be your biggest resource issue when trying to hit those numbers in a multiplayer game though. There's a lot more that will go in to having meaningful gameplay with 100 players.
